okhttp/src/jvmMain/kotlin/okhttp3/Headers.kt

```diff
@@ -180,25 +184,35 @@ actual class Headers internal actual constructor(
      * Add a header with the specified name and formatted date. Does validation of header names and
      * value.
      */
-    fun add(name: String, value: Date) = add(name, value.toHttpDateString())
+    fun add(name: String, value: Date) = apply {
+      add(name, value.toHttpDateString())
+    }
 
     /**
      * Add a header with the specified name and formatted instant. Does validation of header names
      * and value.
      */
-    fun add(name: String, value: Instant) = add(name, Date.from(value))
+    @IgnoreJRERequirement
+    fun add(name: String, value: Instant) = apply {
+      add(name, Date(value.toEpochMilli()))
```

Comment:
Date.from is available if you have already have access to Instant

Reply:
Great point. So maybe this is not a problem at all.

(I assume Animal Sniffer is still broken, as the `@IgnoreJRERequirement` annotations should be required?)

Reply:
Yep. It should be failing since the signature is set to API 21 or something.